posts/inside-rust/2020-JuneJuly-governance-wg-meetings.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Governance Working Group Update: Meeting 21 May 2020"
+author: Val Grimm
+team: The Governance WG <https://github.com/rust-lang/wg-governance>
+---
+
+Hello everyone! 
+
+This blog post summarizes the thematically-related meetings of June 4, June 25, and July 9th. 
+
+## Big picture themes
+Define the role of domain working groups and community groups in the Rust Project; Rust governance taxonomy
+
+You can find the <br>
+[detailed minutes June 4](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.06.04.md)  <br>
+[detailed minutes June 25](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.06.25.md)  <br>
+[detailed minutes July 9](https://github.com/rust-lang/wg-governance/blob/master/minutes/2020.07.09.md)  <br>
+on the [wg-governance](https://github.com/rust-lang/wg-governance) repository, but here is a quick summary: <br>
+
+* There is a need for more clarity about why and how groups are formed, 
+    * what group categories mean, 
+    * how they relate to each other, 
+    * how they relate to the Rust Project
+    * and their level of decision-making power
+* Rust repository structure and permissions should ideally represent status of groups in the project
+* There was discussion around Community Groups having their own repositories outside of the Rust project and not be listed in
+Rust materials. Instead, it is likely that, to avoid some community groups being perceived to be "more official", 
+the Rust project would possibly only link to the [Mozilla Communities Portal](https://community.mozilla.org/), 
+and Rust community groups wishing to do outreach could can create profiles there, share events, and generally do promotion and recruitment
+* "Pre-FCP" likely going away
```

Comment:
```suggestion
* We discussed the idea of streamlining rfcbot procedures, as discussed in rust-lang/wg-governance#38.
```

How about this? I'm not sure people will have any idea what the existing sentence means.